testapi.pm

```diff
@@ -687,7 +621,7 @@ sub assert_script_run {
     }
     my $str = hashed_string("ASR$cmd");
     # call script_run with idle_timeout 0 so we don't wait twice
-    script_run("$cmd; echo $str-\$?- > /dev/$serialdev", 0);
+    script_run("$cmd; echo $str-\$?- | tee /dev/$serialdev", 0);
```

Comment:
Perhaps it is not available on some systems, but I don't like using it regardless, it will probably at least break some needles. I would prefer to simply remove the redirection when using a serial terminal, but I need to know what the current console is. That is not really such a problem though, thinking about it.